lightly/transforms/random_frequency_mask_transform.py

```diff
@@ -0,0 +1,39 @@
+from typing import Tuple
+
+import numpy as np
+import torch
+from torch import Tensor
+
+
+class RFMTransform:
```

Comment:
Let's use the full name as RFM is not a well known abbreviation:

```suggestion
class RandomFrequencyMaskTransform:
```